apidef/oas/middleware.go

```diff
@@ -1274,3 +1274,37 @@ func (e EndpointPostPlugins) ExtractTo(meta *apidef.GoPluginMeta) {
 	meta.PluginPath = e[0].Path
 	meta.SymbolName = e[0].Name
 }
+
+type CircuitBreaker struct {
```

Comment:
@andrei-tyk sorry, unresolving this (sadly on the same line)- struct missing godoc comment,

Maybe we can just link tyk classic apidefs on the CircuitBreaker struct godoc comment and not individual fields, if there's consensus with @jeffy-mathew (per-field migration docs seem excessive).